.meteor/packages

```diff
@@ -56,3 +55,6 @@ kadira:flow-router
 scorpiusjs:config
 scorpiusjs:materialize-admin-only
 useraccounts:bootstrap
+alethes:pages
+kurounin:pagination
+kurounin:pagination-blaze
```

Comment:
OK. No need to reinvent the wheel. I say let's wait until the official release of 1.5 before moving these (and other packages) into dynamic imports.